### PR TITLE
feat: Expose some avatar root APIs

### PR DESCRIPTION
**Pull request by kaikoga**
Related Issue: #49

This PR exposes some utility APIs:

- `RuntimeUtil.IsAvatarRoot()`
- `RuntimeUtil.FindAvatarInParents()`

Also, avatar root determination is defined in a cross platform way:

- With VRCSDK: Treat object with `VRCAvatarDescriptor` as avatar root (unchanged).
- Without VRCSDK: Treat upmost object with `Animator` as avatar root (proposed). Less reliable if some users prefer setup with nesting avatar descriptors upon transplanting outfits (as mentioned in [MA's EasySetupOutfit](https://github.com/bdunderscore/modular-avatar/blob/af7a9c03bae716bc85779c5a6df8ffba07b3ffe3/Packages/nadena.dev.modular-avatar/Editor/EasySetupOutfit.cs#L256-L259)), but we don't want to use platform specific components here.

### CODE REVIEW

File: Runtime/RuntimeUtil.cs [hunk 1]
```diff
@@ -2,7 +2,11 @@
 using System.Collections.Generic;
 using JetBrains.Annotations;
 using UnityEngine;
+using UnityEngine.SceneManagement;
+
+#if VRC_SDK_VRCSDK3
```
Comment (kaikoga) [inline Runtime/RuntimeUtil.cs:7]: Is it okay to add a version define like `NDMF_VRCSDK3_AVATARS` for "com.vrchat.avatars" to asmdef? (naming follows lilToon)

Comment (bdunderscore) [reply Runtime/RuntimeUtil.cs:7]: Yes, that would be fine I think.

Comment (bdunderscore) [reply Runtime/RuntimeUtil.cs:7]: In fact, I think it's necessary - I don't think the defines auto-set by the VRCSDK are a stable API.

**Comment by kaikoga** [comment]
Discussions: https://github.com/bdunderscore/ndmf/discussions/39

**Review by bdunderscore — DISMISSED** [review]

**Comment by bdunderscore** [comment]
Looks like the CHANGELOG check has failed, please add an appropriate changelog entry and repush.

**Review by bdunderscore — APPROVED** [review]

**Pull request merged**